src/libraries/System.DirectoryServices/src/System.DirectoryServices.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
   <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
   <PropertyGroup>
-    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetFramework)' == 'netstandard2.0'">SR.DirectoryServices_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetFramework)' == 'netstandard2.0'">System.SR.DirectoryServices_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
Why do we need `System`?

Reply:
We are not using the using statements, so we need to define the fully qualified name

Reply:
I see. Can we instead append it in the generator if it starts with SR? It seems cumbersome for people to add System to the messages.